packages/graph/lib/deserialize/channel-settings-from-message.ts

```diff
@@ -10,10 +10,10 @@ export function channelSettingsFromMessage(msg: ChannelUpdateMessage): ChannelSe
     instance.direction = msg.direction;
     instance.timestamp = msg.timestamp;
     instance.cltvExpiryDelta = msg.cltvExpiryDelta;
-    instance.htlcMinimumMsat = msg.htlcMinimumMsat;
-    instance.htlcMaximumMsat = msg.htlcMaximumMsat;
-    instance.feeBaseMsat = msg.feeBaseMsat;
-    instance.feeProportionalMillionths = msg.feeProportionalMillionths;
+    instance.htlcMinimumMsat = msg.htlcMinimumMsat.msats;
```

Comment:
Even though this shouldn't be null, let's perform a null check on these properties.  This will allow the test for `GraphManager` to succeed.

Reply:
with null check it worked like a charm